routes/users.js

```diff
@@ -3,10 +3,22 @@ var router = express.Router();
 var db = require("../db.js");
 
 router.route("/")
+  .get(function (req, res) {
+    let user = req.body;
+    db.users.getUserByName(user).then(user => {
```

Comment:
In the readme, you said that this route requires the a "name" object in the JSON body. This line implies that they have to send a complete user object. I would make the db.js route accept a name (instead of the whole user object), and only pass in `user.name` on this line